deploy-changes.sh

```
@@ -0,0 +1,119 @@
+  echo "Error: REMOTE_HOST environment variable is required"
+  exit 1
+fi
+
+if [ -z "$SSH_PASSWORD" ]; then
+  echo "Error: SSH_PASSWORD environment variable is required"
+  exit 1
+fi
+
+SSH_CMD="sshpass -p ${SSH_PASSWORD} ssh -o StrictHostKeyChecking=no ${REMOTE_USER}@${REMOTE_HOST}"
+SCP_CMD="sshpass -p ${SSH_PASSWORD} scp -o StrictHostKeyChecking=no"
+RSYNC_CMD="sshpass -p ${SSH_PASSWORD} rsync -avz -e \"ssh -o StrictHostKeyChecking=no\""
```

Shellcheck (0.10.0)

[warning] RSYNC_CMD appears unused. Verify use (or export if used externally).

(SC2034)